aws/resource_aws_autoscaling_policy_test.go

```diff
@@ -375,3 +415,79 @@ resource "aws_autoscaling_policy" "foobar_simple" {
 }
 `, name, name, name)
 }
+
+func testAccAwsAutoscalingPolicyConfig_TargetTracking_Predefined(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_launch_configuration" "test" {
+  name          = "tf-test-%s"
+  image_id      = "ami-21f78e11"
```

Comment:
Nitpick: Do you mind using the AMI data source here instead, so we don't need to keep updating the ID over time as often?